ci/build.yml

```
@@ -0,0 +1,17 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: onsdigital/dp-concourse-tools-java
+
+inputs:
+  - name: dp-logging
+
+outputs:
+  - name: build
+
```

You may need to add this here:
```suggestion
caches:
  - path: go/
  
```

---

this is a java service so not needed as per other exisiting java services